src/libraries/Common/src/Interop/Windows/SspiCli/SecuritySafeHandles.cs

```diff
@@ -429,36 +429,41 @@ internal abstract partial class SafeDeleteContext : SafeHandle
 
             // Optional output buffer that may need to be freed.
             SafeFreeContextBuffer outFreeContextBuffer = null;
+            Span<IntPtr> ptr = stackalloc IntPtr[3];
```

Comment:
Did you mean to delete this line?

Reply:
yes. The compiler normally warns about unused variables. I'm not sure why this got missed.

Reply:
Maybe because it was stackalloced? Not sure.